vector/src/main/java/im/vector/app/features/createdirect/CreateDirectRoomActivity.kt

```diff
@@ -206,6 +212,12 @@ class CreateDirectRoomActivity : SimpleFragmentActivity() {
     }
 
     private fun renderCreationSuccess(roomId: String) {
+        analyticsTracker.capture(
+                session.getRoomSummary(roomId).toAnalyticsViewRoom(
+                        trigger = ViewRoom.Trigger.MessageUser,
+                        groupingMethod = appStateHandler.getCurrentRoomGroupingMethod()
+                )
+        )
```

Comment:
Can we move this tracking to `navigator.openRoom` impl to avoid code duplication?